src/components/Form/form-validators.js

```diff
@@ -0,0 +1,6 @@
+// export const validateName = (src) => /[A-z]+/.test(src);
+export const validateName = (src) => /^[A-z]+$/.test(src);
+export const validatePassword = (src) => /[A-z]{8,}/.test(src);
+export const validateEmail = (src) => /@/.test(src);
+export const validateLatitude = (src) => +src > -86 || +src < 86;
+export const validateLongitude = (src) => +src > -181 || +src < 181;
```

Comment:
Nice. Here's a chart you can stick on your wall that shows you how unary plus interacts with values vs other methods like parseInt and Number: https://i.stack.imgur.com/LLrgj.png